src/config/product.openshiftContainer.js

```diff
@@ -292,20 +286,36 @@ const config = {
         }),
       isSort: true,
       isWrap: true,
-      width: 20
+      width: 10
+    },
+    {
+      metric: RHSM_API_PATH_METRIC_TYPES.CORES,
+      cell: ({ hasInfiniteCores: hasInfiniteQuantity, [RHSM_API_PATH_METRIC_TYPES.CORES]: total } = {}) => {
+        if (hasInfiniteQuantity === true) {
+          const content = translate(`curiosity-inventory.label`, {
+            context: [SUBSCRIPTIONS_INVENTORY_TYPES.HAS_INFINITE_QUANTITY, RHSM_API_PATH_METRIC_TYPES.CORES]
+          });
+          return (
+            <Tooltip content={content}>
+              <ChartIcon symbol="infinity" size="md" aria-label={content} />
+            </Tooltip>
+          );
+        }
+        return translate('curiosity-inventory.measurement', {
+          context: (total && 'value') || undefined,
+          total,
+          testId: <span data-test={`subscriptions-cell-${RHSM_API_PATH_METRIC_TYPES.CORES}`} data-value={`${total}`} />
+        });
+      },
+      isSort: true,
+      isWrap: true
     },
     {
-      metric: SUBSCRIPTIONS_INVENTORY_TYPES.TOTAL_CAPACITY,
-      header: (data, session, { [INVENTORY_META_TYPES.UOM]: uom } = {}) =>
-        translate('curiosity-inventory.header', { context: ['subscriptions', uom] }),
-      cell: ({
-        [SUBSCRIPTIONS_INVENTORY_TYPES.HAS_INFINITE_QUANTITY]: hasInfiniteQuantity,
-        [SUBSCRIPTIONS_INVENTORY_TYPES.TOTAL_CAPACITY]: total,
-        [SUBSCRIPTIONS_INVENTORY_TYPES.UOM]: uom
-      } = {}) => {
+      metric: RHSM_API_PATH_METRIC_TYPES.SOCKETS,
```

Comment:
regardless of whether we transform the api response to have `Sockets` there is no metric id sorting on the subscriptions api

required: 
- change this back to total_capacity
- or disable the sorting
- or do both